pkg/controller/podgc/gc_controller_test.go

```diff
@@ -714,16 +714,16 @@ func TestGCInspectingPatchedPodBeforeDeletion(t *testing.T) {
 				Status: v1.PodStatus{
 					Phase: v1.PodFailed,
 					Conditions: []v1.PodCondition{
+						{
```

Comment:
curious why this was required

Reply:
(make sure this ordering change isn't flaky / non-deterministic)

Reply:
nice marking as resolved!